gammapy/datasets/make.py

```diff
@@ -64,3 +70,90 @@ def sigma_energy_theta(energy, theta, sigma):
                                        zenith=zenith_hi)
 
     return psf
+
+
+def generate_observation_table(observatory, n_obs):
```

Comment:
You could add ~ a handful of parameters more to this function, e.g. the observatory name or the date range.